src/xdocs/config_filters.xml

```diff
@@ -973,6 +973,143 @@ public class InputTest {
 &lt;suppress-xpath checks=&quot;.*&quot; query=&quot;//METHOD_DEF[@text='legacyMethod']
           /descendant-or-self::node()&quot;/&gt;
         </source>
+        <p>
+          Currently, xpath queries support two types of attributes <code>@text</code> and
+          <code>@value</code>
+          <ul>
+            <li>
+              <code>@text</code> - addresses to the name of the node. For example: class name,
+              method name, variable name and etc.
+            </li>
+            <li>
+              <code>@value</code> - addresses to the value of the node. For example: variable value,
+              annotation value, text content and etc.
+            </li>
+          </ul>
+        </p>
+        <p>For example, to suppress violation on variable <code>wordCount</code> in following
+        code:</p>
+        <source>
+public class InputTest {
+    private int wordCount = 11;
+}
+        </source>
+        <p>You need to look at AST of such code by our CLI tool:</p>
+        <source>
+            $ java -jar checkstyle-X.XX-all.jar -t InputTest.java
+            CLASS_DEF -> CLASS_DEF [1:0]
+            |--MODIFIERS -> MODIFIERS [1:0]
+            |   `--LITERAL_PUBLIC -> public [1:0]
+            |--LITERAL_CLASS -> class [1:7]
+            |--IDENT -> InputTest [1:13]
+            `--OBJBLOCK -> OBJBLOCK [1:23]
+            |--LCURLY -> { [1:23]
+            |--VARIABLE_DEF -> VARIABLE_DEF [2:4]
+            |   |--MODIFIERS -> MODIFIERS [2:4]
+            |   |   `--LITERAL_PRIVATE -> private [2:4]
+            |   |--TYPE -> TYPE [2:12]
+            |   |   `--LITERAL_INT -> int [2:12]
+            |   |--IDENT -> wordCount [2:16]
+            |   |--ASSIGN -> = [2:26]
+            |   |   `--EXPR -> EXPR [2:28]
+            |   |       `--NUM_INT -> 11 [2:28]
+            |   `--SEMI -> ; [2:30]
+            `--RCURLY -> } [3:0]
+        </source>
+        <p>The easiest way is to suppress by variable name. As you can see <code>VARIABLE_DEF</code>
+        node refers to variable declaration statement and has child node with token type
+        <code>IDENT</code> which is used for storing class, method, variable names. Any node can be
+        queried by name, if only it has exactly one direct child with token type
+        <code>IDENT</code>.</p>
```

Comment:
off topic: but this is exact evidence that we did design/naming mistake, as minimum as attribute should be named `ident`.